website/docs/foundations/typography/partials/guidelines/overview.md

```diff
@@ -1,7 +1,7 @@
 
 ## Font stack
 
-Helios uses the default system fonts available on each OS. We opt for system fonts because they are stable and offer various styles that support internationalization, code, tabular data, data viz, etc. In addition, they help improve performance by reducing the number of resources to be loaded, which is important for large applications like those at HashiCorp.
+Helios intentionally uses default system fonts from each operating system, benefiting from their reliability, versatility, and a broad array of styles to cater to diverse needs, including internationalization, code rendering, tabular data, data visualization, and more, ensuring a consistent cross-platform experience, optimizing performance by minimizing additional resource loading, and facilitating smooth and efficient user interactions within HashiCorp's extensive applications.
```

Comment:
```suggestion
Helios relies on the standard fonts provided by each operating system. We choose these fonts because they are reliable and offer a range of styles that support different needs, such as internationalization, code readability, tabular data, and data visualization. Using system fonts helps optimize performance by reducing the number of resources that need to be loaded, and this is particularly beneficial for large applications like those developed by HashiCorp.```